### ISSUE TITLE
[🚀 Feature]: More documentation and examples for `by.py`

### ISSUE THREAD

**Issue by Andre-Medina** [SeleniumHQ/selenium#14746]
### Feature and motivation

Hi all, I've been using Selenium for a few months now. It's amazing, keep up the great work.

I was considering contributing by adding some documentation into the code itself. Particularly to the following classes:
- `py/selenium/webdriver/remote/webdriver.py` to `WebDriver.find_element`
- `py/selenium/webdriver/common/by.py` to `By`

I found these are some of the major components for interacting with webpages and there only documentation on `selenium.dev` which is a bit out of the way. I thinking adding some examples for when different `By` conditions into the code itself would be best. E.g. `By.ID` vs `By.CSS_SELECTOR`.

This documentation will definitely help beginners who are used to python side documentation in their IDE. 

### Usage example

Replicate Documation found on "https://www.selenium.dev/documentation/webdriver/elements/locators/" to the files in `py/selenium/...`

**Comment by shbenzer** [comment]
How are you suggesting these comments be added if not in Selenium.dev? As docstrings?

**Comment by Andre-Medina** [comment]
Yes, in the class docstring, so they can work with interpreters like pylance.

E.g. this description
![image](https://github.com/user-attachments/assets/f5c1fff1-ae14-40ab-887e-7187971aacca)

Comes from the class docstring
![image](https://github.com/user-attachments/assets/dc81d91c-42a9-4a0b-bf5f-b88116527543)

would be very helpful to have some examples and pretty standard across other python packages. For example, in the python package `polars`, the `add` method has the following description:
![image](https://github.com/user-attachments/assets/a3c4988c-1fe0-47b8-9820-0781d266531d)

Which again, comes from the package itself:
![image](https://github.com/user-attachments/assets/b49678ff-2753-4496-a0a5-96aa480ff750)

**Comment by shbenzer** [comment]
I don't have an issue with this. @AutomatedTester @p0deje thoughts?

**Comment by p0deje** [comment]
No objections, let's add those!

**Comment by shbenzer** [comment]
@Andre-Medina We would love for you to contribute to this cause, thank you in advance!

**Comment by Andre-Medina** [comment]
Okay thanks all, I should have some time this week. 

Edit: all good found `CONTRIBUTING.md`

**Comment by p0deje** [comment]
You’d normally fork and send PR from there.

**Comment by Andre-Medina** [comment]
> comment or this will be closed in 14 days

I will try to find some time this coming month
